ntoskrnl/mm/section.c

```diff
@@ -754,7 +758,7 @@ NTSTATUS NTAPI PeFmtCreateSection(IN CONST VOID * FileHeader,
         pssSegments[i].Protection = SectionCharacteristicsToProtect[nCharacteristics >> 28];
         pssSegments[i].WriteCopy = !(nCharacteristics & IMAGE_SCN_MEM_SHARED);
 
-        if(pishSectionHeaders[i].Misc.VirtualSize == 0 || pishSectionHeaders[i].Misc.VirtualSize < pishSectionHeaders[i].SizeOfRawData)
+        if(pishSectionHeaders[i].Misc.VirtualSize == 0)
```

Comment:
Check how this is done in freeldr and in Alex's bootlib:
https://git.reactos.org/?p=reactos.git;a=blob;f=boot/freeldr/freeldr/lib/peloader.c;hb=895dccd4ac7cc9c8dcb4a679e34fe0b50c560b2e#l819
https://git.reactos.org/?p=reactos.git;a=blob;f=boot/environ/lib/misc/image.c;hb=895dccd4ac7cc9c8dcb4a679e34fe0b50c560b2e#l967
(Basically, if SizeOfRawData > VirtualSize then set SizeOfRawData = VirtualSize). Indeed, the raw data (and its size) size might have been rounded up to the next section alignment, but we know that its meaningful contents, when loaded in memory, must not be larger than VirtualSize.

Reply:
I'm not sure what change you're suggesting, if any.
My tests cover `SizeOfRawData > VirtualSize`, `SizeOfRawData < VirtualSize`, `SizeOfRawData = 0`, and they all pass with the code as presented in this PR.

Reply:
Just showing how the sections loop contents should look like...